db/re/skill_db.yml

```diff
@@ -17911,6 +17911,72 @@ Body:
         NoReiteration: true
         NoOverlap: true
     Status: PropertyWalk
+  - Id: 751
+    Name: NPC_ALL_STAT_DOWN
+    Description: Decrease All Stats
+    MaxLevel: 5
+    TargetType: Attack
+    DamageFlags:
+      NoDamage: true
+    Flags:
+      IsNpc: true
+    Range: 9
+    Duration1: 10000
+    Status: ALL_STAT_DOWN
+  - Id: 752
+    Name: NPC_GRADUAL_GRAVITY
+    Description: Increased Gravity
+    MaxLevel: 3
+    TargetType: Attack
+    DamageFlags:
+      NoDamage: true
+    Flags:
+      IsNpc: true
+    Range: 9
+    Duration1: 3000
+    Status: GRADUAL_GRAVITY
+  - Id: 753  
+    Name: NPC_DAMAGE_HEAL
+    Description: Damage Heal
+    MaxLevel: 3
+    TargetType: Self
+    DamageFlags:
+      NoDamage: true
+    Flags:
+      IsNpc: true
+    Duration1: 30000
+    Status: DAMAGE_HEAL
+  - Id: 754
+    Name: NPC_IMMUNE_PROPERTY
+    Description: Property Immune
+    MaxLevel: 10 #Check: Max level is 5?
```

Comment:
according to kRO's official current lub it is level 5.
```c
	[SKID.NPC_IMMUNE_PROPERTY] = {
		"NPC_IMMUNE_PROPERTY",
		SkillName = "이뮨 엘리멘탈",
		MaxLv = 5,
		SpAmount = { 0, 0, 0, 0, 0 },
		bSeperateLv = false,
		AttackRange = { 5, 10, 15, 20, 25 }
	},

Reply:
I fixed max level, but actually to apply this, skill mechanism must be fixed first. 
I have no idea how to implement right now...

Without completion of the skill it would be better leave max level to 10.